prometheus-metric-storage/src/lib.rs

```diff
@@ -17,6 +18,8 @@ use std::sync::Mutex;
 pub use prometheus::{Error, Opts, Registry, Result};
 
 #[doc(hidden)]
+// nlordell: Any reason why these are hidden? I would expect this to have link
+// to the procedural macro docs.
```

Comment:
No reason, situation with docs was a bit chaotic at the time of publishing this crate. It is almost fixed now, and this entry will not be hidden when I'm done 😉